apps/i18n/music/en_us.json

```diff
@@ -38,6 +38,9 @@
   "blockly_fieldRestBeats": "{num} beats",
   "blockly_fieldRestOneMeasure": "1 measure",
   "blockly_fieldRestMeasures": "{num} measures",
+  "blockly_fieldTriggerStartImmediately": "immediately",
+  "blockly_fieldTriggerStartNextBeat": "at next beat",
+  "blockly_fieldTriggerStartNextMeasure": "at next measure",
```

Comment:
Mildly curious how this these fragments (and the containing block) will localize...